### DISCUSSION TITLE
platform/actors/development/programming-interface/standby

### DISCUSSION THREAD

**Discussion by giscus** [apify/apify-docs#1134] [Comments]
# platform/actors/development/programming-interface/standby

Use the Actor as a real-time API server.

https://docs.apify.com/platform/actors/development/programming-interface/standby

**Comment by dimitramuni** [comment]
Is it possible to modify the existing apify actor, e.g. apify/website-content-crawler in Standby Mode?

**Comment by lionfreezer59-art** [comment]
It's nice
